include/pros/adi.hpp

```diff
@@ -39,8 +39,6 @@ class ADIPort {
 	 */
 	ADIPort(std::uint8_t port, adi_port_config_e_t type = E_ADI_TYPE_UNDEFINED);
 
-	virtual ~ADIPort(void);
```

Comment:
I think this should still have a virtual default dtor.

Reply:
Ya, you're right